src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,61 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName, true))) {
+            String[] array = reader(fromFileName);
+            String[] reportArray = reportGenerator(array);
+            for (String reportLine : reportArray) {
+                bufferedWriter.write(reportLine);
+            }
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("Can`t find a file", e);
+        } catch (IOException e) {
+            throw new RuntimeException("Line can`t be written", e);
+        }
+    }
+
+    private String[] reader(String fileName) {
```

Comment:
```suggestion
    private String[] readFromFile(String fileName) {
```
назва методу повинна починатись з дієслова